samples/react/package.json

```diff
@@ -108,7 +108,7 @@
     "build:client:rendering-host": "cross-env-shell PUBLIC_URL=$npm_package_config_tunnelUrl \"react-scripts build\"",
     "build:server": "cross-env-shell NODE_ENV=production \"webpack --config server/server.webpack.config.js\"",
     "bootstrap": "node scripts/bootstrap.js",
-    "graphql:update": "node -r babel-register ./scripts/update-graphql-fragment-data.js",
```

Comment:
Looks like these react sample changes accidentally snuck in here?

Reply:
Yeah haha I was just trying to revert this 😳